RACK-Ontology/OwlModels/.gitignore

```diff
@@ -0,0 +1,40 @@
+/AGENTS.owl
```

Comment:
Please remove this file.  We don't need it since 1) ignoring OWL files already in git doesn't work, and 2) when the time comes, we'll remove OWL files from git and put *.owl in RACK/.gitignore at the root level.